### PR TITLE
fix(perf): get current commit tag in CI

### PR DESCRIPTION
**Pull request by miroslavstastny**
Fix the way how we get a tag of a current commit in CI.
Changelog left blank.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@ jobs:
           name: Save Statistics to DB
           command: |
             if [ -n "${STATS_URI}" ]; then
-              yarn stats:save --tag=`git tag --points-at`
+              yarn stats:save --tag=`git tag --points-at HEAD`
```
Comment (miroslavstastny) [inline .circleci/config.yml:85]: It does not work without the `HEAD` in CI

**Review by bmdalex — APPROVED** [review]

File: build/gulp/tasks/stats.ts [hunk 2]
```diff
@@ -148,7 +148,10 @@ function readCurrentBundleStats() {
 }
 
 task('stats:save', async () => {
-  const commandLineArgs = _.pick(argv, ['sha', 'branch', 'tag', 'pr', 'build'])
+  const commandLineArgs = _.pickBy(
+    _.pick(argv, ['sha', 'branch', 'tag', 'pr', 'build']),
+    val => val !== '', // ignore empty strings
```
Comment (layershifter) [inline build/gulp/tasks/stats.ts:153]: NIT: 
```suggestion
    _.identity, // ignore empty strings
```

**Review by layershifter — APPROVED** [review]

File: build/gulp/tasks/stats.ts (hunk 2 above)
Comment (miroslavstastny) [reply build/gulp/tasks/stats.ts:153]: The reason why I haven't used `_.idenitity` is that it removes all falsy values:

```js
_.pickBy({stringVal: '', numberVal: 0, booleanVal: false}, _.identity)
{}

_.pickBy({stringVal: '', numberVal: 0, booleanVal: false}, val => val !== '')
{numberVal: 0, booleanVal: false}
```

So if you want to be able to specify `--build=0` on command line, you cannot use `_.identity`.

Comment (layershifter) [reply build/gulp/tasks/stats.ts:153]: 👍

**Pull request merged**